packages/tldraw/src/state/shapes/StickyUtil/StickyUtil.tsx

```diff
@@ -126,6 +127,7 @@ export class StickyUtil extends TDShapeUtil<T, E> {
 
       // Resize to fit text
       React.useEffect(() => {
+        console.log('hi!');
```

Comment:
Please remove this console.log, it looks like you forget this here

Reply:
Hey, this PR isn't ready for merging. Sorry for the confusion. I add **THIS PR IS NOT READY FOR MERGING** to the PR description. That's in there for debugging reasons at the moment, and I'll remove when I get closer to being ready for a final review.